monkey/monkey_island/windows/create_certificate.bat

```diff
@@ -20,13 +20,13 @@ copy "%mydir%windows\openssl.cfg" "%mydir%bin\openssl\openssl.cfg"
 
 :: Change file permissions
 SET adminsIdentity="BUILTIN\Administrators"
-FOR /f %%O IN ('whoami') DO SET ownIdentity=%%O
+FOR /f "delims=''" %%O IN ('whoami') DO SET ownIdentity=%%O
 
 FOR %%F IN ("%mydir%cc\server.key", "%mydir%cc\server.csr", "%mydir%cc\server.crt") DO (
 
 	:: Remove all others and add admins rule (with full control)
-	echo y| cacls %%F" /p %adminsIdentity%:F
+	echo y| cacls %%F /p "%adminsIdentity%":F
```

Comment:
Why do you need quotes around `adminsIdentity`? It's a constant set in line 22 which should work fine without the quotes.